webui/react/src/components/ExperimentStopModal.tsx

```diff
@@ -24,6 +24,9 @@ interface Props {
 
 const ExperimentStopModalComponent: React.FC<Props> = ({ experimentId, onClose }: Props) => {
   const [type, setType] = useState<AvalableActions>(ActionType.Cancel);
+  const f_flat_runs = useFeature().isOn('flat_runs');
+
+  const buttonText = f_flat_runs ? 'Stop Search' : 'Stop Experiment';
```

Comment:
sugg: you could make this variable lowercase, use it as is in `publicSubject` and the confirmation copy, and then `capitalize` it for the button text.